### ISSUE TITLE
Graphic Bugs in Adventure Island 2 and 4

### ISSUE THREAD

**Issue by Papermanzero** [libretro/nestopia#77]
The intro screen is completely messed up.

**Comment by hizzlekizzle** [comment]
Do you have the NstDatabase.xml file in your system directory?

**Comment by Papermanzero** [comment]
Yes, I have it in the system directory.
Do I need to create a folder.
The documentation only states to put the xml in the system directory.

**Comment by hizzlekizzle** [comment]
no, just putting it in there should be good enough. I'll try to reproduce the behavior this evening.

**Comment by retro-wertz** [comment]
even without nstdatabase, i dont have gfx bug at least for Adventure Island II, you must have a bad dump ( where did get Adventure Island IV?

**Comment by Papermanzero** [comment]
I switched to mesen, and with mesen it is working.

**Comment by retro-wertz** [comment]
that sounds like you don't have the correct header then in that rom then,
and it means its not nestopia is not able to find NstDatabase. at least for
Adventure Island II, it should be there in database-if you had it in the
correct folder.

**Comment by rdanbrook** [comment]
Can you paste the CRC or SHA1 hash of the rom without the header?
They should be:
CRC: 771C8855
SHA1: D30741F41F76B34C08032FC61579DC5963F1443D

I can't reproduce the problem you're experiencing.
